src/exposure-provider.tsx

```diff
@@ -197,12 +199,13 @@ export const ExposureProvider: React.FC<ExposureProviderProps> = ({
     async function checkSupportAndStart() {
       await supportsExposureApi();
 
+      await configure();
```

Comment:
Moved this line out of the condition as the job still runs for users with enf off / unsupported.

Reply:
You should not call configure before isReady is true, this will result in invalid config being passed to the module, please remove this change. For example app has not yet loaded users refresh and auth token so calling configure too soon will result in empty tokens being stored in shared prefs. 
The background process checks for valid config when it runs so not calling config is not an issue

Reply:
reverted. instead move the serverUrl config check further up